Analyzer/interface/CommonFunction.h

```diff
@@ -1609,7 +1609,8 @@ reco::DeDxData computedEdx(const float& track_eta,
         bool specInCPE = (isOnEdge || hasBadPixels || spansTwoROCs) ? true : false;
 
         if (specInCPE) continue;
-        if (probQ>0.8) continue;
+//        20 March 2023 : no cut on individual probQ for Ih computation
+//        if (probQ>0.8) continue;
```

Comment:
so this should be kept in general

Reply:
I am not sure about this. Clearly this individual probQ cut introduces a bias on Ih, as the pixel clusters with small deposit are systematically removed from the Ih computation. So it was not clear on Mattermost if Morris recommends not to apply this cut for Ih... I can revert it...

Reply:
So Morris recommended that we use ProbQ for the final Ih values. But when you are doing the scale factor to match the pixel and the strips, then the full distributions should be considered (w/o the probQ cut)

Reply:
ok. see update of the PR below 
[put back individual probQ cut for Ih and modification of trigger sele…](https://github.com/CMS-HSCP/SUSYBSMAnalysis-HSCP/pull/129/commits/ef7ace5326511d44ef5a573aca8f166b60d8ceb7)
[ef7ace5](https://github.com/CMS-HSCP/SUSYBSMAnalysis-HSCP/pull/129/commits/ef7ace5326511d44ef5a573aca8f166b60d8ceb7)